paypal/functions/index.js

```diff
@@ -0,0 +1,108 @@
+var functions = require('firebase-functions');
```

Comment:
let's make it all ES2015 style code like the other samples. Can you replace the `var` with `const` ?